.teamcity/settings.kts

```diff
@@ -351,6 +353,15 @@ object RunAllUnitTests : BuildType({
 		}
 		perfmon {
 		}
+		pullRequests {
+			vcsRootExtId = "${WpCalypso.id}"
+			provider = github {
+				authType = token {
+					token = "credentialsJSON:57e22787-e451-48ed-9fea-b9bf30775b36"
```

Comment:
This seems sensitive. Should we have the token set up as a repository secret?

Reply:
It is already a secret, a "TeamCity" one. We are using the option "[Store secure values outside VCS](https://www.jetbrains.com/help/teamcity/storing-project-settings-in-version-control.html#Storing+Secure+Settings)", which 

> TeamCity stores a randomly generated IDs in XML configuration files instead of the scrambled passwords. Actual passwords are stored on the disk under the TeamCity Data Directory and are not checked into the version control system.

Reply:
Nice 👍  Thanks for elaborating